editor/plugins/path_3d_editor_plugin.h

```diff
@@ -113,6 +128,7 @@ class Path3DEditorPlugin : public EditorPlugin {
 
 	static Path3DEditorPlugin *singleton;
 	virtual EditorPlugin::AfterGUIInput forward_3d_gui_input(Camera3D *p_camera, const Ref<InputEvent> &p_event) override;
+	virtual void forward_3d_draw_over_viewport(Control *p_overlay) override;
```

Comment:
```suggestion
```

If you don't need it, remove it.

Reply:
Sure.

I add this method early because I want to draw some part of gizmo with 2d overlay lines. But that requires the transform of the viewport camera which is not directly available in `forward_3d_draw_over_viewport`. Do you have any suggestions?

Reply:
I simply asked for this to be removed because there was no content in `forward_3d_draw_over_viewport()` in cpp.

Drawing 2D objects in a 3D viewport is a difficult, but shaders can be used to do some of it. I don't know if that helps in drawing 2D lines, but Skeleton3DGizmo uses a hack of the Shader to enforce the drawing order.

Reply:
Then I'll just remove the method for now, and left it for future. Thanks for the info.